### PR TITLE
NT change template ids

### PR DESCRIPTION
**Pull request by sammhmrc**

### CODE REVIEW

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/twowaymessage/twoWayMessagingTemplates.scala [hunk 1]
```diff
@@ -23,7 +23,7 @@ import uk.gov.hmrc.hmrcemailrenderer.templates.ServiceIdentifier.TwoWayMessaging
 object twoWayMessagingTemplates {
   val templates = Seq(
     MessageTemplate.create(
-      templateId = "twoWayMessageReceived",
+      templateId = "2wsm-customer",
       fromAddress = FromAddress.noReply("twoWayMessingRecieved"),
```
Comment (sammhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/twowaymessage/twoWayMessagingTemplates.scala]: @matthewgroom what should this be?

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/twowaymessage/twoWayMessagingTemplates.scala [hunk 2]
```diff
@@ -32,7 +32,7 @@ object twoWayMessagingTemplates {
       priority = Some(MessagePriority.Background)
```
Comment (sammhmrc) [inline app/uk/gov/hmrc/hmrcemailrenderer/templates/twowaymessage/twoWayMessagingTemplates.scala]: @matthewgroom What should this be?

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/twowaymessage/twoWayMessagingTemplates.scala (hunk 1 above)
Comment (matthewgroom) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/twowaymessage/twoWayMessagingTemplates.scala]: It's perfect as it is. As long as the 'templateId' is the same name as the one inside the 'ServiceIdentifier' file, then it's good to go

File: app/uk/gov/hmrc/hmrcemailrenderer/templates/twowaymessage/twoWayMessagingTemplates.scala (hunk 2 above)
Comment (matthewgroom) [reply app/uk/gov/hmrc/hmrcemailrenderer/templates/twowaymessage/twoWayMessagingTemplates.scala]: Background is correct

**Pull request merged**